client/src/components/Related/ProductCard.jsx

```diff
@@ -1,42 +1,66 @@
 import React, { useState, useEffect } from 'react';
 import ratingsAPI from '../../API/Ratings.js';
+import { useNavigate } from 'react-router-dom';
 
 const RelatedProduct = (props) => {
+  const navigate = useNavigate();
   const [averageRating, setAverageRating] = useState(0);
+  const stars = props.generateStars(averageRating, 'related');
 
+  // const isInYourOutfit = () => {
+  //   return props.yourOutfit.filter((product) => product.id === props.product.id).length > 0
+  // }
+
+  const routeChange = () => {
```

Comment:
Function to change the URL to the new product ID when a related product is clicked. Should update state in App.jsx to new product ID and pass down to the other components.

Reply:
Nothing major to change here, but here are my thoughts --
- possibly move useEffect up to line 9 so it's clear how state is being updated upon mounting of this component
- get rid of the console.log on line 16
- move the "?" on line 46 to it's own line below like you did with the ":" on line 51 so it's easier to follow the ternary

Things you did well --
- your name conventions are really clear for variables and class names
- your functions are small and easy to follow